cmd/gzv/cli/console.go

```diff
@@ -176,6 +176,86 @@ func (c *nonceCmd) parse(args []string) bool {
 	return true
 }
 
+type voteMinerPoolCmd struct {
+	gasBaseCmd
+	addr string
+}
+
+func genVoteMinerPoolCmd() *voteMinerPoolCmd {
+	c := &voteMinerPoolCmd{
+		gasBaseCmd: *genGasBaseCmd("voteMinerPool", "only guard miner node can vote miner pool,every guard miner node only vote once"),
```

Comment:
only guard miner node can for vote miner pool, each guard miner node can only vote once